integration-tests/jpa-postgresql/src/main/java/io/quarkus/it/jpa/postgresql/Person.java

```diff
@@ -64,8 +71,25 @@ public void setStatus(Status status) {
         this.status = status;
     }
 
+    /**
+     * Need to explicitly set the scale (and the precision so that the scale will actually be read from the annotation).
+     * Postgresql would only allow maximum scale of 6 for a `interval second`.
+     *
+     * @see org.hibernate.type.descriptor.sql.internal.Scale6IntervalSecondDdlType
+     */
+    @Column(precision = 5, scale = 5)
+    @JdbcType(PostgreSQLIntervalSecondJdbcType.class)
```

Comment:
I think this is supposed to be simply `@JdbcType(INTERVAL_SECOND)`? See https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html#_duration

Reply:
That's what I thought looking at the docs, too 😃, but the annotation wants a class rather than an int 🙈. Should we update that to be `@JdbcType(INTERVAL_SECOND_JDBC_TYPE_CLASS)` or something similar?

Reply:
It should be `@JdbcTypeCode(INTERVAL_SECOND)`, I believe. And yes it would be nice to send a PR to update the docs.

Reply:
ok will test it locally and then send an update to the docs, thanks!

Reply:
Any news on this one? It looks like you didn't update the PR, is it because `@JdbcTypeCode` didn't work?

Reply:
The `@JdbcTypeCode` lead to https://hibernate.atlassian.net/browse/HHH-16591 🙈 
And since someone picked up that ticket to work on, I thought I'd give it a chance to see if it gets fixed 😃 
The missing parts are: 
- this annotation, 
- to address a comment from Sanne (https://github.com/quarkusio/quarkus/pull/33300#discussion_r1199034326) to simplify the registering a bit

Right?

Reply:
Right.

Reply:
Regarding my previous comment.. it's a nice to have but definitely not a requirement ;)